<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomColorsPropertiesView</class>
 <widget class="QWidget" name="CustomColorsPropertiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SliderWidget" name="sliderWidgetBrushSize" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="buttonLoadTexture">
     <property name="text">
      <string>Load Texture</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSaveTexture">
     <property name="text">
      <string>Save Texture</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>225</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWidget</class>
   <extends>QWidget</extends>
   <header>SliderWidget/SliderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
